.github/dependabot.yaml

```diff
@@ -0,0 +1,24 @@
+version: 2
+
+registries:
+  etterlatte-libs:
+    type: maven-repository
+    url: https://maven.pkg.github.com/navikt/pensjon-etterlatte-libs
+    username: x-access-token
+    password: "${{secrets.NAV_READ_PACKAGES}}"
+
+updates:
+  - package-ecosystem: gradle
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "04:00"
+    open-pull-requests-limit: 15
```

Comment:
Det er både nice, og litt unice. Digg å ikke spamme ned _helt_, men samtidig litt frustrerende å resolve èn PR bare for å se at det dukker opp en ny. Men med så mange i teamet bør vi nok ha den på for å ikke begrave alle andre PRer. Tror nesten jeg ville satt den lavere på bakgrunn av dèt resonnementet